server/src/routes/twiml.ts

```diff
@@ -1,20 +1,28 @@
 import { Request, RequestHandler, Response } from 'express';
 import { twiml, validateExpressRequest } from 'twilio';
 import { ServerCredentials } from '../common/types';
+import { error } from '../utils/log';
 
 export function createTwimlRoute(
   serverConfig: ServerCredentials,
 ): RequestHandler {
   const { VoiceResponse } = twiml;
```

Comment:
Preference: bind an error handler here.
```
const logError = (msg: string) => {
  error('/twiml ${msg}`);
};
```
Later you can do something like
```
const msg = 'Missing "To".';
logError(msg);
res.status(400).send(msg);
```
So we're less prone to having different return values and error logs.